### PR TITLE
Fixes #24807 - unsafe html in toast notification

### PR DESCRIPTION
**Pull request by sharvit**
toast notification sends strings through as HTML

@tbrisker

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const AlertBody = ({
 
     {title && <strong>{title}</strong>}
 
-    <span dangerouslySetInnerHTML={{ __html: message }} />
+    {message}
```
Comment (ohadlevy) [inline webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js:13]: i believe this was here to allow rails to send formatted html? would that still work?

Comment (sharvit) [reply webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js:13]: I haven't found any code doing that.

@tbrisker showed me this issue https://projects.theforeman.org/issues/24807 and we see it problematic in some way.

We decided that once we will find those places in the UI we will fill a new bug and fix them.

Comment (tbrisker) [reply webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js:13]: The helpers we use actually escape any HTML already. If any notification is found sending HTML we should fix it.

**Review by tbrisker — CHANGES_REQUESTED** [review]
Looks like a bunch of js tests failed, some snapshots need updating.

**Comment by sharvit** [comment]
Thanks @tbrisker, snapshots updated.

**Review by tbrisker — APPROVED** [review]

**Comment by mmoll** [comment]
[test foreman]
[test upgrade]
[test katello]

**Review by tbrisker — APPROVED** [review]
Thanks @sharvit !

**Comment by tbrisker** [comment]
Cherrypicked to 1.19 as b586c053e381fedd0767adb2618959dbc4cb3a51

**Comment by tbrisker** [comment]
@sharvit can you open a cherry-pick pr against 1.18-stable please?

**Pull request merged**